src/canvas.tsx

```diff
@@ -193,7 +195,7 @@ export const Canvas = React.memo(
     useEffect(() => {
       state.current.aspect = size.width / size.height || 0
 
-      if (state.current.camera.isOrthographicCamera) {
+      if (state.current.camera instanceof THREE.OrthographicCamera) {
```

Comment:
would prefer isOrthographicCamera here b/c it's the official way. the idea is that cams can be both perspective and orthographic, the flag indicates which is active

Reply:
That's fair, I think this was returning a type error because `isOrthographicCamera` is not a defined property on `THREE.Camera` or `THREE.PerspectiveCamera`.

This was the way I settled on to satisfy TypeScript, but the alternative might be to typecast `state.current.camera` like the following:

```
if ((state.current.camera as THREE.OrthographicCamera).isOrthographicCamera)
```